presto-main/src/main/java/com/facebook/presto/cost/ComparisonStatsCalculator.java

```diff
@@ -27,53 +28,64 @@
 import static java.lang.Double.POSITIVE_INFINITY;
 import static java.lang.Double.isNaN;
 
-public final class ComparisonStatsCalculator
+public class ComparisonStatsCalculator
 {
-    private ComparisonStatsCalculator() {}
+    private ComparisonStatsCalculator()
+    {}
 
-    public static PlanNodeStatsEstimate comparisonSymbolToLiteralStats(
+    public static PlanNodeStatsEstimate comparisonExpressionToLiteralStats(
```

Comment:
You could keep the old method as:
```
    public static PlanNodeStatsEstimate comparisonSymbolToLiteralStats(
            PlanNodeStatsEstimate inputStatistics,
            Symbol symbol,
            OptionalDouble doubleLiteral,
            ComparisonExpressionType type) {
        return comparisonExpressionToLiteralStats(
                inputStatistics,
                Optional.of(symbol),
                inputStatistics.getSymbolStatistics(symbol),
                doubleLiteral,
                type);
    }
```

Reply:
Is there a point in keeping the old method?